src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java

```diff
@@ -103,4 +105,29 @@ public static AudioItem resolveItem(Player player, String input) throws IllegalA
         return item.get();
     }
 
+    public static String colourFormat(Color color) {
+        return String.format("#%02X%02X%02X", color.getRed(), color.getGreen(), color.getBlue());
+    }
+
+    public static Color getColor(String string) {
+        for (Field f : Color.class.getFields()) {
+            try {
+                if (f.getType() == Color.class && ((f.getModifiers() & java.lang.reflect.Modifier.FINAL) == java.lang.reflect.Modifier.FINAL) && f.getName().equalsIgnoreCase(string)) {
```

Comment:
What's the point? Why not import Modifier? Why not use the functions already provided by Color? Also, reflection is slow. It does everything at run-time making it extremely slow.